xbmc/pvr/PVRDatabase.cpp

```diff
@@ -21,6 +21,7 @@
 #include "PVRDatabase.h"
 
 #include <utility>
+#include <xbmc/pvr/channels/PVRChannelGroup.h>
```

Comment:
Sure, this is something that CLion generated automatically. Will change to standard quotes.